tasks_downstream.py

```diff
@@ -183,3 +183,28 @@ def logs(c, tail=10):
         cmd += f" --tail {tail}"
     with c.cd(str(PROJECT_ROOT)):
         c.run(cmd)
+
+
+@task(develop)
+def psql(c, db=None):
+    """Get a psql"""
+    cmd = "docker-compose run --rm odoo psql"
+    if db:
+        cmd += f" {db}"
+    c.run(cmd)
+
+
+@task(develop)
+def shell(c, db=None):
+    """Get an Odoo shell"""
+    cmd = "docker-compose run --rm odoo odoo shell"
+    if db:
+        cmd += f" -d {db}"
+    c.run(cmd)
+
+
+@task(develop)
+def scaffold(c, name):
+    """Create a scaffold"""
+    cmd = f"docker-compose run --rm odoo odoo scaffold {name} /opt/odoo/custom/src/private"
```

Comment:
this is mounted as readonly because that's how it will be in production.

Maybe you can just add a `-v` to this `docker-compose run`?

Also TBH I personally will not use this. I find https://github.com/OCA/maintainer-tools/tree/master/template/module to be much more complete and I just copy-paste it. A proper copier template for a module would be best IMHO.